docs/examples/hypervisors/kvm/README.md

```diff
@@ -0,0 +1,105 @@
+KVM Virtualization
+==================
+
+This sample uses Terraform to demonstrate the Bring Your Own Hypervisor model to deploy custom KVM based Virtual Machines on top of a KVM hypervisor on Oracle Cloud Infrastructure.
```

Comment:
Looks like this covers some of the same territory as https://docs.us-phoenix-1.oraclecloud.com/Content/Resources/Assets/installing_kvm_multi_vnics.pdf - would it make sense to provide a link?

Reply:
That's correct, all the scripts are based on the official documentation and that white paper. Yes, it make sense to provide the link.

Reply:
I just updated the README file including your suggestions.

Reply:
The primary (maybe only) use case for BYO-KVM is supporting OS’es the platform doesn’t support natively. This will generally be older OS’es and third-party appliances. If we’re going to provide a working example we should do so end-to-end and in support of that use case. We shouldn’t get into the details of the use case of the guest, it’s confusing and appears to limit the applicability of BYO-KVM.

The readme should be something like - 
###
The Bring Your Own KVM (BYO-KVM) capability of OCI is designed to support customers who need to run operating systems and third-party appliances that the platform doesn’t support natively. For a list of natively supported OS’es see https://docs.us-phoenix-1.oraclecloud.com/Content/Compute/References/images.htm. 

In this example we demonstrate the automatic installation of Oracle Linux 4.x as guest VM on a Bare Metal instance running KVM in Oracle Cloud Infrastructure (OCI). It can be easily adapted to automate the installation of any OS that KVM supports, see https://www.linux-kvm.org/page/Guest_Support_Status._
###